packages/api/Dockerfile

```
@@ -84,8 +86,10 @@ COPY patches ./patches
 COPY scripts ./scripts
 COPY packages/docker-git-session-sync ./packages/docker-git-session-sync
 COPY packages/lib ./packages/lib
+COPY packages/terminal ./packages/terminal
 
 RUN bun run --cwd packages/docker-git-session-sync build
+RUN bun run --cwd packages/terminal build
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)